ui/src/utils/truncate.test.ts

```diff
@@ -0,0 +1,17 @@
+import { truncate } from './truncate';
+
+describe('truncate', () => {
+  it(`should truncate a long string`, () => {
+    const text = 'Who looks outside, dreams; who looks inside, awakes.';
+    const obtainedText = truncate(text, 26);
+
+    expect(obtainedText).toBe('Who looks outside, dreams...');
+  });
+
+  it(`should truncate a long string`, () => {
```

Comment:
Is this test description correct? Should it be something like `it('does not truncate a string shorter than the specified length')`?

Reply:
Yeah exactly, bad copy & paste job there - I knew I needed tests for the tests :)